### DISCUSSION TITLE
[Buildings] Understanding file tiling scheme

### DISCUSSION THREAD

**Discussion by jfmartinez4** [OvertureMaps/data#195] [Discussion]
### Category Feedback

Add details 
 I downladed the Buildings parquet files and wanted to understand how the buildings were partitioned into the files. Whether it's by latitude and longitude tiles, or if it's another schema.

**Comment by jwass** [comment]
Hi @jfmartinez4 The buildings dataset is more or less sorted by the geohash 15. The rows within a file are sorted the same. It was decent enough for a first cut and gives some reasonable separation to improve spatial query performance but could probably be better.

I'll close this out but feel free to open or make a new issue with further questions
